lib/validators/spec/openapi.js

```diff
@@ -0,0 +1,244 @@
+const util = require('../../util');
```

Comment:
This file is mostly identical to `lib/validators/spec/swagger.js` there's just some minor differences to account for the different specs (mostly surrounding `components` vs `definitions`).

Reply:
It's pretty tough for me to parse out what's new functionality in this file but lots of exciting stuff in here 🤩